src/unlimited.h

```diff
@@ -186,32 +200,33 @@ extern std::set<uint256> setUnVerifiedOrphanTxHash;
 extern CCriticalSection cs_xval;
 // Xpress Validation: end
 
-extern void LoadFilter(CNode *pfrom, CBloomFilter *filter);
+extern void LoadFilter(CNode* pfrom, CBloomFilter* filter);
 
-extern bool CheckAndRequestExpeditedBlocks(CNode* pfrom);  // Checks to see if the node is configured in bitcoin.conf to be an expedited block source and if so, request them.
-extern void SendExpeditedBlock(CXThinBlock& thinBlock,unsigned char hops, const CNode* skip=NULL);
-extern void SendExpeditedBlock(const CBlock& block,const CNode* skip=NULL);
-extern void HandleExpeditedRequest(CDataStream& vRecv,CNode* pfrom);
+extern bool CheckAndRequestExpeditedBlocks(CNode* pfrom); // Checks to see if the node is configured in bitcoin.conf to
+                                                          // be an expedited block source and if so, request them.
+extern void SendExpeditedBlock(CXThinBlock& thinBlock, unsigned char hops, const CNode* skip = NULL);
+extern void SendExpeditedBlock(const CBlock& block, const CNode* skip = NULL);
+extern void HandleExpeditedRequest(CDataStream& vRecv, CNode* pfrom);
 extern bool IsRecentlyExpeditedAndStore(const uint256& hash);
 
 
-extern CSemaphore*  semOutboundAddNode;
-extern std::vector<CNode*> xpeditedBlk; // Who requested expedited blocks from us
+extern CSemaphore* semOutboundAddNode;
+extern std::vector<CNode*> xpeditedBlk;   // Who requested expedited blocks from us
 extern std::vector<CNode*> xpeditedBlkUp; // Who we requested expedited blocks from
 extern std::vector<CNode*> xpeditedTxn;
-extern CStatHistory<uint64_t > recvAmt; 
-extern CStatHistory<uint64_t > sendAmt; 
+extern CStatHistory<uint64_t> recvAmt;
+extern CStatHistory<uint64_t> sendAmt;
 
 // Connection Slot mitigation - used to track connection attempts and evictions
 struct ConnectionHistory
 {
-    double nConnections;      // number of connection attempts made within 1 minute
-    int64_t nLastConnectionTime;  // the time the last connection attempt was made
+    double nConnections;         // number of connection attempts made within 1 minute
+    int64_t nLastConnectionTime; // the time the last connection attempt was made
```

Comment:
This will smear diffs over many unrelated lines. If I rename nLastConnectionTime to something shorter for instance, the formatter will change nConnections, which is unwanted.

Because of the other screw up related to same line comment, that'd be great if we adopted as a policy to not have comment on the same line as what is commented but the line before. Even better if there is a way to teach this to clang-format, but comments are notoriously hard to format properly, so maybe that's not possible.